src/super_gradients/examples/predict/detection_predict.py

```diff
@@ -2,12 +2,12 @@
 from super_gradients.training import models
 
 # Note that currently only YoloX and PPYoloE are supported.
-model = models.get(Models.PP_YOLOE_S, pretrained_weights="coco")
+model = models.get(Models.DECIYOLO_L, pretrained_weights="coco")
 
 IMAGES = [
     "https://miro.medium.com/v2/resize:fit:500/0*w1s81z-Q72obhE_z",
-    "https://s.hs-data.com/bilder/spieler/gross/128069.jpg",
     "https://datasets-server.huggingface.co/assets/Chris1/cityscapes/--/Chris1--cityscapes/train/28/image/image.jpg",
+    "https://media.timeout.com/images/105921973/750/422/image.jpg",
 ]
-prediction = model.predict(IMAGES, iou=0.65, conf=0.5)
+prediction = model.predict(IMAGES, iou=0.65, conf=0.6)
```

Comment:
This is problematic, we are really letting our users figure out for themselves what visually works out the best...

Reply:
Default added in d974313215196cc3c4210f1c49682040b93285cf